### ISSUE TITLE
operator error

### ISSUE THREAD

**Issue by cecil1313** [zhanghang1989/PyTorch-Encoding#164]
Hi,  when i try to "import encoding". I got the error like this. Could you help me with that, thanks :)

Error building extension 'enclib_gpu': [1/2] c++ -MMD -MF operator.o.d -DTORCH_EXTENSION_NAME=enclib_gpu -DTORCH_API_INCLUDE_EXTENSION_H -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/torch/csrc/api/include -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/TH -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/THC -isystem /usr/local/cuda/include -isystem /home/liucheng/miniconda2/envs/py36/include/python3.6m -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++11 -c /home/liucheng/PyTorch-Encoding/encoding/lib/gpu/operator.cpp -o operator.o
FAILED: operator.o 
c++ -MMD -MF operator.o.d -DTORCH_EXTENSION_NAME=enclib_gpu -DTORCH_API_INCLUDE_EXTENSION_H -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/torch/csrc/api/include -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/TH -isystem /home/liucheng/miniconda2/envs/py36/lib/python3.6/site-packages/torch/lib/include/THC -isystem /usr/local/cuda/include -isystem /home/liucheng/miniconda2/envs/py36/include/python3.6m -D_GLIBCXX_USE_CXX11_ABI=0 -fPIC -std=c++11 -c /home/liucheng/PyTorch-Encoding/encoding/lib/gpu/operator.cpp -o operator.o
/home/liucheng/PyTorch-Encoding/encoding/lib/gpu/operator.cpp:3:16: error: expected constructor, destructor, or type conversion before ‘(’ token
 PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
                ^
ninja: build stopped: subcommand failed.

**Comment by ytoon** [comment]
I implement a simple Synchronized BatchNorm, which is compatible with PyTorch 1.0 and inspired by this project. You can try my code.

**Comment by cecil1313** [comment]
Thank you :) 
I will try it.

**Comment by tjulyz** [comment]
@cecil1313 How do you solve this problem?

**Comment by tjulyz** [comment]
@cecil1313  I just found that add #include <pybind11/pybind11.h> at the top of operator.cpp helps solve this problem.
